ken_nguyen/assignments/Ruby OOP/bankAccount.rb

```diff
@@ -0,0 +1,69 @@
+#bank account
+
+class BankAccount
+  attr_accessor :account_number, :checking, :savings
+
+  @@interest_rate = 0.06 #is it still private if its up here?
+
+  @@accounts = 0
+
+  FLAG = [:checking, :savings]
+
+  def initialize
+    @account_number = rand_account_number
+    @checking = 0
+    @savings = 0
+    @@accounts += 1
+  end
+
+  def checking_bal
+    puts @checking
+  end
+
+  def savings_bal
+    puts @savings
+  end
+
+  def deposit(val, flag)
+    case
```

Comment:
A better way is to do

case flag
when :checking
when :savings

== is implied on case statements